ballerina/api_inspectionlot_srv/modules/mock/api_inspectionlot_srv_mock_service.bal

```diff
@@ -0,0 +1,77 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/os;
+
+boolean isBalBuild = os:getEnv("IS_BAL_BUILD") == "true";
+string certPathPostFix = isBalBuild ? "../" : "/home/ballerina/ballerina/";
+
+listener http:Listener ep0 = new (9090,
+    secureSocket = {
+        key: {
+            certFile: certPathPostFix + "resources/public.crt",
+            keyFile: certPathPostFix + "resources/private.key"
+        }
+    }
+);
+
+service /sap/opu/odata/sap/API_INSPECTIONLOT_SRV on ep0 {
+
+resource function head .() returns http:Response {
```

Comment:
Lets format file

Reply:
Formatted here [Format mock server and tests](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/11/commits/ad23f904e41a597186cd124ea9f291599f3ff112)